### ISSUE TITLE
Keyboard focus does not return to trigger after closing menu

### ISSUE THREAD

**Issue by graphicidentity** [MeD-DMC/Adapt-accessibilityfixes#161]
After selecting "Course Resources" or "Page Progress" the learner is taking "outside" to a slide-out pane. Upon selecting close on the either slide-out pane, the keyboard focus moves several stops earlier to the "Skip to Main Content" button. I believe this is a failure of WCAG 2.4.3 Focus Order.
